scss/_variables.scss

```diff
@@ -118,6 +118,13 @@ $enable-pointer-cursor-for-buttons:           true !default;
 $enable-print-styles:                         true !default;
 $enable-validation-icons:                     true !default;
 
+// Show deprecation messages
+//
+// Some mixins and functions will be removed in v5, the `deprecate()` mixin will warn you if you're using one of these.
+// These warnings can be suppressed by setting this variable to `false`:
+
+$show-deprecation-messages:                   true !default;
```

Comment:
Should we stick with `$enable-` here?

Also, this is set to true, but shouldn't it be set fo `false` to start? This replaces `$ignore-warning != true`, so it's disabled to start I believe.